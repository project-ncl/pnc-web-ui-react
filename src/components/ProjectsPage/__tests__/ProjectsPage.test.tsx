import { render } from '@testing-library/react';
import { ProjectsPage } from '../ProjectsPage';

test('renders ProjectsPage', () => {
  // render(<ProjectsPage />);
});
