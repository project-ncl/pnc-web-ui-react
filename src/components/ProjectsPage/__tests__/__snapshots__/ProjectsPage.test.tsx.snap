// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display ProjectsPage component compare snapshot with previous record 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="pf-c-page__main-section pf-m-light"
      >
        <div
          class="pf-c-content"
        >
          <h1
            class=""
            data-ouia-component-id="OUIA-Generated-Text-3"
            data-ouia-component-type="PF4/Text"
            data-ouia-safe="true"
            data-pf-content="true"
          >
            Projects
          </h1>
          <p
            class=""
            data-ouia-component-id="OUIA-Generated-Text-4"
            data-ouia-component-type="PF4/Text"
            data-ouia-safe="true"
            data-pf-content="true"
          >
            This page contains a standalone projects like 
            <span
              class="pf-c-label"
            >
              <span
                class="pf-c-label__content"
              >
                Hibernate
              </span>
            </span>
             or 
            <span
              class="pf-c-label"
            >
              <span
                class="pf-c-label__content"
              >
                JBoss Modules
              </span>
            </span>
            , usually a project represents one SCM repository and one project may contain multiple Build Configs.
          </p>
        </div>
      </section>
      <div
        class="pf-c-divider"
        role="separator"
      />
      <section
        class="pf-c-page__main-section"
      >
        <table
          aria-label="Projects List"
          class="pf-c-table pf-m-grid-md pf-m-compact pf-m-no-border-rows"
          data-ouia-component-id="OUIA-Generated-Table-4"
          data-ouia-component-type="PF4/Table"
          data-ouia-safe="true"
          role="grid"
        >
          <thead
            class=""
          >
            <tr
              class=""
              data-ouia-component-id="OUIA-Generated-TableRow-9"
              data-ouia-component-type="PF4/TableRow"
              data-ouia-safe="true"
            >
              <th
                class="pf-m-width-30"
                data-key="0"
                data-label="Name"
                scope="col"
              >
                Name
              </th>
              <th
                class=""
                data-key="1"
                data-label="Description"
                scope="col"
              >
                Description
              </th>
              <th
                class="pf-m-width-15"
                data-key="2"
                data-label="Build Configs count"
                scope="col"
              >
                Build Configs count
              </th>
            </tr>
          </thead>
          <tbody
            class=""
            role="rowgroup"
          >
            <tr
              class=""
              data-ouia-component-id="OUIA-Generated-TableRow-10"
              data-ouia-component-type="PF4/TableRow"
              data-ouia-safe="true"
            >
              <td
                class=""
                data-key="0"
                data-label="Name"
              >
                <a
                  href="/"
                >
                  Test Project Name 100
                </a>
              </td>
              <td
                class=""
                data-key="1"
                data-label="Description"
              >
                Test Project Name 100 Desc
              </td>
              <td
                class=""
                data-key="2"
                data-label="Build Configs count"
              >
                0
              </td>
            </tr>
            <tr
              class=""
              data-ouia-component-id="OUIA-Generated-TableRow-11"
              data-ouia-component-type="PF4/TableRow"
              data-ouia-safe="true"
            >
              <td
                class=""
                data-key="0"
                data-label="Name"
              >
                <a
                  href="/"
                >
                  zz_2021110-101
                </a>
              </td>
              <td
                class=""
                data-key="1"
                data-label="Description"
              >
                zz_2021110-101 desc
              </td>
              <td
                class=""
                data-key="2"
                data-label="Build Configs count"
              >
                0
              </td>
            </tr>
            <tr
              class=""
              data-ouia-component-id="OUIA-Generated-TableRow-12"
              data-ouia-component-type="PF4/TableRow"
              data-ouia-safe="true"
            >
              <td
                class=""
                data-key="0"
                data-label="Name"
              >
                <a
                  href="/"
                >
                  zz_2021110-102
                </a>
              </td>
              <td
                class=""
                data-key="1"
                data-label="Description"
              />
              <td
                class=""
                data-key="2"
                data-label="Build Configs count"
              >
                0
              </td>
            </tr>
            <tr
              class=""
              data-ouia-component-id="OUIA-Generated-TableRow-13"
              data-ouia-component-type="PF4/TableRow"
              data-ouia-safe="true"
            >
              <td
                class=""
                data-key="0"
                data-label="Name"
              >
                <a
                  href="/"
                >
                  zz_2021110-103
                </a>
              </td>
              <td
                class=""
                data-key="1"
                data-label="Description"
              />
              <td
                class=""
                data-key="2"
                data-label="Build Configs count"
              >
                0
              </td>
            </tr>
            <tr
              class=""
              data-ouia-component-id="OUIA-Generated-TableRow-14"
              data-ouia-component-type="PF4/TableRow"
              data-ouia-safe="true"
            >
              <td
                class=""
                data-key="0"
                data-label="Name"
              >
                <a
                  href="/"
                >
                  zz_2021110-104
                </a>
              </td>
              <td
                class=""
                data-key="1"
                data-label="Description"
              />
              <td
                class=""
                data-key="2"
                data-label="Build Configs count"
              >
                0
              </td>
            </tr>
            <tr
              class=""
              data-ouia-component-id="OUIA-Generated-TableRow-15"
              data-ouia-component-type="PF4/TableRow"
              data-ouia-safe="true"
            >
              <td
                class=""
                data-key="0"
                data-label="Name"
              >
                <a
                  href="/"
                >
                  zz_2021110-105
                </a>
              </td>
              <td
                class=""
                data-key="1"
                data-label="Description"
              >
                zz_2021110-105 description
              </td>
              <td
                class=""
                data-key="2"
                data-label="Build Configs count"
              >
                0
              </td>
            </tr>
            <tr
              class=""
              data-ouia-component-id="OUIA-Generated-TableRow-16"
              data-ouia-component-type="PF4/TableRow"
              data-ouia-safe="true"
            >
              <td
                class=""
                data-key="0"
                data-label="Name"
              >
                <a
                  href="/"
                >
                  Test Project Name 106
                </a>
              </td>
              <td
                class=""
                data-key="1"
                data-label="Description"
              >
                Test Project Name 106 description
              </td>
              <td
                class=""
                data-key="2"
                data-label="Build Configs count"
              >
                17
              </td>
            </tr>
          </tbody>
        </table>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="pf-c-page__main-section pf-m-light"
    >
      <div
        class="pf-c-content"
      >
        <h1
          class=""
          data-ouia-component-id="OUIA-Generated-Text-3"
          data-ouia-component-type="PF4/Text"
          data-ouia-safe="true"
          data-pf-content="true"
        >
          Projects
        </h1>
        <p
          class=""
          data-ouia-component-id="OUIA-Generated-Text-4"
          data-ouia-component-type="PF4/Text"
          data-ouia-safe="true"
          data-pf-content="true"
        >
          This page contains a standalone projects like 
          <span
            class="pf-c-label"
          >
            <span
              class="pf-c-label__content"
            >
              Hibernate
            </span>
          </span>
           or 
          <span
            class="pf-c-label"
          >
            <span
              class="pf-c-label__content"
            >
              JBoss Modules
            </span>
          </span>
          , usually a project represents one SCM repository and one project may contain multiple Build Configs.
        </p>
      </div>
    </section>
    <div
      class="pf-c-divider"
      role="separator"
    />
    <section
      class="pf-c-page__main-section"
    >
      <table
        aria-label="Projects List"
        class="pf-c-table pf-m-grid-md pf-m-compact pf-m-no-border-rows"
        data-ouia-component-id="OUIA-Generated-Table-4"
        data-ouia-component-type="PF4/Table"
        data-ouia-safe="true"
        role="grid"
      >
        <thead
          class=""
        >
          <tr
            class=""
            data-ouia-component-id="OUIA-Generated-TableRow-9"
            data-ouia-component-type="PF4/TableRow"
            data-ouia-safe="true"
          >
            <th
              class="pf-m-width-30"
              data-key="0"
              data-label="Name"
              scope="col"
            >
              Name
            </th>
            <th
              class=""
              data-key="1"
              data-label="Description"
              scope="col"
            >
              Description
            </th>
            <th
              class="pf-m-width-15"
              data-key="2"
              data-label="Build Configs count"
              scope="col"
            >
              Build Configs count
            </th>
          </tr>
        </thead>
        <tbody
          class=""
          role="rowgroup"
        >
          <tr
            class=""
            data-ouia-component-id="OUIA-Generated-TableRow-10"
            data-ouia-component-type="PF4/TableRow"
            data-ouia-safe="true"
          >
            <td
              class=""
              data-key="0"
              data-label="Name"
            >
              <a
                href="/"
              >
                Test Project Name 100
              </a>
            </td>
            <td
              class=""
              data-key="1"
              data-label="Description"
            >
              Test Project Name 100 Desc
            </td>
            <td
              class=""
              data-key="2"
              data-label="Build Configs count"
            >
              0
            </td>
          </tr>
          <tr
            class=""
            data-ouia-component-id="OUIA-Generated-TableRow-11"
            data-ouia-component-type="PF4/TableRow"
            data-ouia-safe="true"
          >
            <td
              class=""
              data-key="0"
              data-label="Name"
            >
              <a
                href="/"
              >
                zz_2021110-101
              </a>
            </td>
            <td
              class=""
              data-key="1"
              data-label="Description"
            >
              zz_2021110-101 desc
            </td>
            <td
              class=""
              data-key="2"
              data-label="Build Configs count"
            >
              0
            </td>
          </tr>
          <tr
            class=""
            data-ouia-component-id="OUIA-Generated-TableRow-12"
            data-ouia-component-type="PF4/TableRow"
            data-ouia-safe="true"
          >
            <td
              class=""
              data-key="0"
              data-label="Name"
            >
              <a
                href="/"
              >
                zz_2021110-102
              </a>
            </td>
            <td
              class=""
              data-key="1"
              data-label="Description"
            />
            <td
              class=""
              data-key="2"
              data-label="Build Configs count"
            >
              0
            </td>
          </tr>
          <tr
            class=""
            data-ouia-component-id="OUIA-Generated-TableRow-13"
            data-ouia-component-type="PF4/TableRow"
            data-ouia-safe="true"
          >
            <td
              class=""
              data-key="0"
              data-label="Name"
            >
              <a
                href="/"
              >
                zz_2021110-103
              </a>
            </td>
            <td
              class=""
              data-key="1"
              data-label="Description"
            />
            <td
              class=""
              data-key="2"
              data-label="Build Configs count"
            >
              0
            </td>
          </tr>
          <tr
            class=""
            data-ouia-component-id="OUIA-Generated-TableRow-14"
            data-ouia-component-type="PF4/TableRow"
            data-ouia-safe="true"
          >
            <td
              class=""
              data-key="0"
              data-label="Name"
            >
              <a
                href="/"
              >
                zz_2021110-104
              </a>
            </td>
            <td
              class=""
              data-key="1"
              data-label="Description"
            />
            <td
              class=""
              data-key="2"
              data-label="Build Configs count"
            >
              0
            </td>
          </tr>
          <tr
            class=""
            data-ouia-component-id="OUIA-Generated-TableRow-15"
            data-ouia-component-type="PF4/TableRow"
            data-ouia-safe="true"
          >
            <td
              class=""
              data-key="0"
              data-label="Name"
            >
              <a
                href="/"
              >
                zz_2021110-105
              </a>
            </td>
            <td
              class=""
              data-key="1"
              data-label="Description"
            >
              zz_2021110-105 description
            </td>
            <td
              class=""
              data-key="2"
              data-label="Build Configs count"
            >
              0
            </td>
          </tr>
          <tr
            class=""
            data-ouia-component-id="OUIA-Generated-TableRow-16"
            data-ouia-component-type="PF4/TableRow"
            data-ouia-safe="true"
          >
            <td
              class=""
              data-key="0"
              data-label="Name"
            >
              <a
                href="/"
              >
                Test Project Name 106
              </a>
            </td>
            <td
              class=""
              data-key="1"
              data-label="Description"
            >
              Test Project Name 106 description
            </td>
            <td
              class=""
              data-key="2"
              data-label="Build Configs count"
            >
              17
            </td>
          </tr>
        </tbody>
      </table>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
